gdx/src/com/badlogic/gdx/FadingGame.java

```diff
@@ -0,0 +1,199 @@
+/*******************************************************************************
+ * Copyright 2011 See AUTHORS file.
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *	 http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ ******************************************************************************/
+
+package com.badlogic.gdx;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import com.badlogic.gdx.graphics.Pixmap;
+import com.badlogic.gdx.graphics.g2d.Batch;
+import com.badlogic.gdx.graphics.glutils.FrameBuffer;
+import com.badlogic.gdx.utils.transition.ScreenTransition;
+import com.badlogic.gdx.utils.transition.TransitionListener;
+
+/** An {@link Game} that delegates to a {@link Screen}. Allows to apply different transitions to screens. </p>
+ * <p>
+ * Screens are not disposed automatically. You must handle whether you want to keep screens around or dispose of them when another
+ * screen is set.
+ * @author iXeption */
+public class FadingGame extends Game {
+
+	protected FrameBuffer currentScreenFBO;
+	protected FrameBuffer nextScreenFBO;
+	protected Screen nextScreen;
+	protected Batch batch;
+
+	private List<TransitionListener> listerns;
```

Comment:
Typo. Also this could be final and initialized right here or in the constructor, I see no reason for it to be initialized in create(). And a minor detail, something like clearTransitionListeners() could be useful.